dev-raku/App-Prove6/App-Prove6-0.0.12-r2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit rakudo
+
+DESCRIPTION="Run tests through a TAP harness."
+HOMEPAGE="https://raku.land/cpan:LEONT/App::Prove6"
+SRC_URI="mirror://cpan/authors/id//L/LE/LEONT/Perl6/${P}.tar.gz"
```

Comment:
Could we/should we use the same SRC_URI magic as perl-module.eclass?

Reply:
There are currently 3 different package ecosystems for raku.

I just read perl-module.eclass which already has 6 variables for handling cpan.

Beyond CPAN, raku also accepts packages from arbitrary git repositories and zef package ecosystem.

There is no algorithm for automatically generating SRC_URI for arbitrary git repositories.

I don't know how rakudo.eclass can automatically generate SRC_URI for zef package ecosystem. Zef download URLs include random numbers for each version of each package.

Reply:
OK, fair enough.